starknet/src/types/proposal.cairo

```diff
@@ -1,6 +1,9 @@
 use starknet::{ContractAddress, storage_access::StorePacking, Store, contract_address_const};
 use sx::{
-    utils::math::pow, types::{FinalizationStatus, UserAddress, user_address::UserAddressTrait}
+    utils::math::pow,
+    types::{
+        FinalizationStatus, UserAddress, user_address::{UserAddressTrait, ContractAddressDefault}
```

Comment:
we re-exported `user_address` so shouldnt need the full import path

Reply:
022db4eec4c3cb91d24b7f99eb679ea1771c2039